guessing-game-v2/contracts/GuessingGame.sol

```diff
@@ -22,45 +22,49 @@ contract GuessingGame {
 
     constructor() {
         owner = msg.sender;
-        _resetSecretNumber();
+        _setSecretNumber();
     }
 
-    function guess(uint256 _number) external payable {
-        require(_number > 0 && _number <= MAX_GUESS, 'Secret number should be between 1 and 1000');
-        require(msg.value == GUESS_FEE, 'You need to pay a fees of 0.443 ETH to make a guess');
-        totalGuesses += 1;
+function guess(uint256 _number) external payable {
+    require(_number > 0 && _number <= MAX_GUESS, 'Secret number should be between 1 and 1000');
+    require(msg.value == GUESS_FEE, 'You need to pay a fees of 0.443 ETH to make a guess');
+    totalGuesses += 1;
 
-        if (lastGuess[msg.sender] == 0 || lastResetTime > lastGuess[msg.sender]) {
-            lastGuess[msg.sender] = 0;
-            lastDifference[msg.sender] = 0;
-        }
+    if (lastGuess[msg.sender] == 0 || lastResetTime > lastGuess[msg.sender]) {
+        lastDifference[msg.sender] = MAX_GUESS + 1;
+    }
 
-        uint256 currentDifference = _number > secretNumber ? _number - secretNumber : secretNumber - _number;
-        string memory feedback;
+    uint256 currentDifference = _number > secretNumber ? _number - secretNumber : secretNumber - _number;
+    string memory feedback;
 
-        if (_number == secretNumber) {
-            payable(msg.sender).transfer(address(this).balance);
-            feedback = "correct! You won the prize";
-            emit Guessed(msg.sender, _number, true, feedback);
-            _resetSecretNumber();
-            totalGuesses = 0;
-        } else {
-            if (lastGuess[msg.sender] != 0) {
-                if (currentDifference < lastDifference[msg.sender]) {
-                    feedback = "warmer";
-                } else if (currentDifference > lastDifference[msg.sender]) {
-                    feedback = "colder";
-                } else {
-                    feedback = "neither warmer nor colder";
-                }
+    if (_number == secretNumber) {
+        payable(msg.sender).transfer(address(this).balance);
+        feedback = "correct! You won the prize";
+        emit Guessed(msg.sender, _number, true, feedback);
+        _resetSecretNumber();
+        totalGuesses = 0;
+    } else {
+        if (lastGuess[msg.sender] != 0 && lastResetTime <= lastGuess[msg.sender]) {
```

Comment:
Boss, the sugested change to if (lastResetTime > lastGuess[msg.sender]) would disrupt the game's feedback mechanism. It would result in players only receiving feedback if the secret number was reset after their last guess  eliminating feedback for current game session guesses